vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageTextItem.kt

```diff
@@ -138,13 +139,19 @@ abstract class MessageTextItem : AbsMessageItem<MessageTextItem.Holder>() {
         val previewUrlView by bind<PreviewUrlView>(R.id.messageUrlPreview)
         private val richMessageStub by bind<ViewStub>(R.id.richMessageTextViewStub)
         private val plainMessageStub by bind<ViewStub>(R.id.plainMessageTextViewStub)
-        var richMessageView: AppCompatTextView? = null
+        var richMessageView: EditorStyledTextView? = null
             private set
         var plainMessageView: AppCompatTextView? = null
             private set
 
         fun requireRichMessageView(): AppCompatTextView {
-            val view = richMessageView ?: richMessageStub.inflate().findViewById(R.id.messageTextView)
+            val view = richMessageView ?: richMessageStub.inflate().findViewById<EditorStyledTextView>(R.id.messageTextView).also {
+                // Required to ensure that `inlineCodeBgHelper` and `codeBlockBgHelper` are initialized
+                it.updateStyle(
+                        styleConfig = it.styleConfig,
+                        mentionDisplayHandler = null,
+                )
+            }
```

Comment:
@jmartinesp let me know what you think about this fix.

Reply:
This is a bit weird, `updateStyle` should automatically be called when it's attached to a window. Are we settings the text in this component before that happens? If so, then yes, we need to add this workaround.

Reply:
Yes, this seems to be the case. It's alright then.

Reply:
I have put a breakpoint in `EditorStyledTextView.onAttachedToWindow()` but was not reaching it before the app crashes.